fix: Bug Report: Tone Dropdown Not (visually) Initializing with Corre…

syuzanvs opened the pull request:
…ct Tone from localStorage

- Updated the tone dropdown menu UI to reflect the active tone loaded from localStorage upon initialization.
- Ensured that the checkmark appears next to the previously selected tone after refreshing the page.
- Modified the initialization logic in the `AiAgentToneCommand` class to set the active tone in the dropdown based on the stored value.

Closes #124

jjroelofs reviewed (changes requested):
When I check out your branch, it switches back to storing the full tone of voice string instead of the label. We can't use the full value because it is content that some users (like me) update frequently. The label is a more reliable reference for the tone of voice, see my commit here:


https://github.com/dxpr/ckeditor5-ai-agent/commit/d12bc0b894b42936c2d03ea3a1dbfb2ae128ef3e

syuzanvs committed:
fix: Bug Report: Tone Dropdown Not (visually) Initializing with Correct Tone from localStorage

- Updated the tone dropdown menu UI to reflect the active tone loaded from localStorage upon initialization.
- Ensured that the checkmark appears next to the previously selected tone after refreshing the page.
- Modified the initialization logic in the AiAgentToneCommand class to set the active tone in the dropdown based on the stored value

Closes #124

syuzanvs commented:
> When I check out your branch, it switches back to storing the full tone of voice string instead of the label. We can't use the full value because it is content that some users (like me) update frequently. The label is a more reliable reference for the tone of voice, see my commit here:
> 
> [d12bc0b](https://github.com/dxpr/ckeditor5-ai-agent/commit/d12bc0b894b42936c2d03ea3a1dbfb2ae128ef3e)

@jjroelofs  I did it like that as we would have an issue with translation when they select another language for the website. Bu now I have added a key, so based on it everything now works correct, pls review

jjroelofs reviewed (changes requested):
great
